### ISSUE TITLE
QuasiQuoter-"Problem"

### ISSUE THREAD

**Issue by jvoigtlaender** [fmidue/modelling-tasks#203]
Liegt dies: 
![image](https://github.com/jvoigtlaender/modelling-tasks-internal/assets/5853832/1d389436-4069-40d7-a6d6-756f8c201c76)

irgendwie an einem falschen QuasiQuoter in https://github.com/jvoigtlaender/modelling-tasks-internal/blob/a1db41f93a445ea246cbc4a4cc275812bdd25f25/src/Modelling/PetriNet/Conflict.hs#L329-L336?

Haben die englischen Formulierungen das gleiche Problem?

**Comment by marcellussiegburg** [comment]
Ja, das liegt an der Wahl des QuasiQuoters (bzw. an einem fehlenden, forcierten Leerzeichen). Das Problem tritt bei der englischen Variante nicht auf.
